strawberry-daiquiri.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "<Strawberry Daiquiri>",
```

Comment:
```suggestion
    "name": "Strawberry Daiquiri",
```

Please remove all the `<`  and `>` in this file as they are just token placeholders.

Reply:
I have made all the changes. Added the picture and the keywords. Sorry for the previous comments, I am very new to this. Also, how would I know if my pull request gets accepted?

Reply:
You'll know when it gets accepted when it gets merged.